generate/domains.txt

```diff
@@ -959,8 +999,16 @@ glendale.net
 globalfree.it
 globalpagan.com
 globalsite.com.br
+gmai.com
+gmail.co
+gmail.co.uk
 gmail.com
+gmail.com.com
+gmail.com1
```

Comment:
[This is an invalid email address](https://travis-ci.org/derhuerst/email-providers/jobs/169609186#L158). Will submit another PR.